spec/services/order_details/reconciler_spec.rb

```diff
@@ -12,7 +12,7 @@
     end
   end
   let(:params) { order_details.each_with_object({}) { |od, h| h[od.id.to_s] = ActionController::Parameters.new(reconciled: "1") } }
-  let(:reconciler) { described_class.new(OrderDetail.all, params, Time.current) }
+  let(:reconciler) { described_class.new(OrderDetail.all, params, Time.current, "reconciled") }
```

Comment:
Are you adding these just to be explicit, right?  The specs would pass without this?

Reply:
It's now a required param. It can either be `"reconciled"` or `nil`